<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyListWidgetItem</class>
 <widget class="QWidget" name="MyListWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_path">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>49</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_test">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_image</zorder>
  <zorder>label_path</zorder>
  <zorder>label_test</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
